docs/hub/api.md

```diff
@@ -462,6 +462,22 @@ Get the daily papers curated by AK and the community. It's the equivalent of [ht
 
 To filter on a particular date, simply pass the date like so: https://huggingface.co/api/daily_papers?date=2025-03-31.
 
+Other keywords that are supported: 
+
+1. `limit` - the number of pages returned
+2. `p` - the page number, for paging through results
+3. `submitter` - to filter by summitter
+4. `date`, `week`, `month` - for filtering by date range
```

Comment:
```suggestion
4. `date`, `week`, `month` - for filtering by date range
```
of the paper release dates?

Reply:
According to [DeepWiki](https://deepwiki.com/search/for-this-api-call-to-filter-on_8c11b99a-0261-4015-b0d6-a2c10e7302eb?mode=fast)

The date parameter refers to when the paper was included in the daily papers on the Hub, not when it was written or originally submitted to arXiv. hf_api.py:1527-1528

This is confirmed by the PaperInfo dataclass, which has two distinct date fields:

published_at: The date the paper was published (on arXiv) hf_api.py:1513-1514
submitted_at: The date the paper appeared in daily papers on the Hub hf_api.py:1527-1528|

I will update it according to that.